rotkehlchen/chain/zksync_lite/manager.py

```diff
@@ -438,6 +439,32 @@ def _query_zksync_api_transactions(
             from_hash = result[-1]['txHash']
             transactions = []
 
+    def query_single_transaction(
+            self,
+            tx_hash: EVMTxHash,
+            concerning_address: ChecksumEvmAddress,
+    ) -> ZKSyncLiteTransaction | None:
+        """Queries zksync lite api for a single transaction, saves it
+        in the DB and returns it if existing
+
+        In case of error returns None and logs the error.
+        """
+        response = self._query_api(url=f'transactions/{tx_hash.hex()}/data')
+        if (tx_entry := response.get('tx', None)) is None:
+            log.error(f'Could not find {tx_hash.hex()} transaction from zksync lite api. Response: {response}')  # noqa: E501
+            return None
+
+        if (tx := self._deserialize_zksync_transaction(entry=tx_entry, concerning_address=concerning_address)) is None:  # noqa: E501
```

Comment:
There are some cases where `_deserialize_zksync_transaction` returns None without logging the reason. I believe we should add a log statement there